app/views/organizations/adopter_fosterer/fostered_pets/index.html.erb

```diff
@@ -13,7 +13,13 @@
                                   class: 'rounded-circle', 
                                   style: "width: 100%; height: 100%; object-fit: cover;") %>
                   </div>
-                  <h5 class="card-title mb-0"><%= fostered_pet.pet.name %></h5>
+                  <h5 class="card-title mb-0 flex-grow-1" style="margin-right: 15px;"><%= fostered_pet.pet.name %></h5>
+                  <div class="fostered-dates ms-auto">
+                    <div class="d-flex flex-column">
+                      <h5 class="card-title mb-1">Term:</h5>
```

Comment:
I actually prefer what you showed where you had Start date: and End date: as I think it looked better and was more descriptive!